wscript

```diff
@@ -39,7 +39,7 @@ def options(opt):
 def configure(conf):
   env = conf.env
 
-  env.append_unique('CXXFLAGS', ['-O2', '-Wall', '-g', '-pipe', '-fno-omit-frame-pointer', '-pthread'])
+  env.append_unique('CXXFLAGS', ['-O2', '-Wall', '-g', '-pipe', '-fno-omit-frame-pointer', '-pthread', '-msse4.2'])
```

Comment:
We discussed offline and we decided NOT to specify `'-msse4.2' by default.
So users needs to build by themselves to benefit from SSE4.2.